iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -424,6 +426,12 @@ private CloseableIterable<T> openGeneric(FileScanTask task, Schema readSchema) {
           iterable = newOrcIterable(inputFile, task, readSchema);
           break;
         case PARQUET:
+          if (inputFile instanceof HadoopInputFile) {
```

Comment:
What's the change meaning? Is it related the replace tag?

Reply:
ohh shit, was something else running internally & it got added accidentally in this, I will remove in the next iter